frontend/apps/docs/content/docs/index.mdx

```diff
@@ -4,6 +4,7 @@ description: Liam ERD is a tool that effortlessly generates beautiful and easy-t
 ---
 
 import { Package, Globe } from "lucide-react";
+import { Tab, Tabs } from 'fumadocs-ui/components/tabs'; // For package-install code blocks
```

Comment:
I considered adding an ignore comment for Biome, but given that the Biome check was successful, I opted to keep only the comment.

https://github.com/liam-hq/liam/blob/8305af4b543df264775be0cb1ea4ae1af4ad0239/frontend/apps/docs/biome.jsonc#L9